### DISCUSSION TITLE
Using wix.exe to sign wix3 bundle

### DISCUSSION THREAD

**Discussion by tm-nti** [wixtoolset/issues#7671] [Questions] [answered]
This question is somewhat related to #7606 

We have decided to downgrade just the bundle to wix v3 but when we sign we use the new dotnet wix tool to detach/reattach the engine

Is this a bad idea? It seems to work fine

**Comment by robmen** [comment, marked as answer]
Yeah, that seems like a bad idea. If it works for now, I wouldn't necessarily expect it to keep working. It might even silently stop working.
